app/component/search/search-input.cjsx

```diff
@@ -106,14 +104,11 @@ class SearchInput extends React.Component
       items={@state?.suggestions || []}
       renderItem={(item) ->
         <SuggestionItem ref={item.name} item={item} spanClass="autosuggestIcon"/>}
-      getSuggestionValue={(suggestion) ->
-        SuggestionItem.getName(suggestion.properties)
-      }
       onSuggestionSelected={@currentItemSelected}
       focusedItemIndex={@state.focusedItemIndex}
       inputProps={
-        "id": "autosuggest-input"
-        "value": @state?.value || ""
+        "id": @props.id
+        "value": if @state.value?.length >= 0 then @state?.value else @props.initialValue
         "onChange": @handleUpdateInputNow
         "onKeyDown": @handleOnKeyDown
         "placeholder": @context.getStore('SearchStore').getPlaceholder()
```

Comment:
Good question! According to design, there is no placeholder in the search. So that is superfluous and can be deleted